api4/emoji.go

```diff
@@ -28,6 +30,8 @@ func (api *API) InitEmoji() {
 }
 
 func createEmoji(c *Context, w http.ResponseWriter, r *http.Request) {
+	defer io.Copy(ioutil.Discard, r.Body)
```

Comment:
0/5, but I'm wondering if there's a way to generalize this. It stands out as "why does this API call discard body" and not others?

Reply:
I don't see a general solution as a release bugfix patch. I would create a ticket to solve it later in master.